build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1997,6 +1997,22 @@ message CacheCapabilities {
   // [BatchUpdateBlobs][build.bazel.remote.execution.v2.ContentAddressableStorage.BatchUpdateBlobs]
   // requests.
   repeated Compressor.Value supported_batch_update_compressors = 7;
+
+  // Maximum size a single CAS entry can have in the remote cache system.
+  // Here is how implementations should interpret this value:
+  // - If it is 0, it means there is no limit is set. Arbitrarily large blobs
+  //   may be uploaded to and downloaded from the cache.
+  // - If it is larger than 0, implementations *should* not attempt to upload
+  //   blobs with size larger than the limit. If they do, it is up to the
+  //   cache implementation what to do, e.g. accept the request anyway or reject
+  //   it as invalid.
+  //   If the cache implementation return a given limit, it *may* still serve
+  //   blobs larger than this limit. This is again implementation defined behaviour.
+  //
+  // Overall, treat this limit as a hint from the cache. The limit may be set for
+  // performance reasons mainly, since when limit is known beforehand some upload
+  // requests can then be skipped completely and avoid roundtrips.
+  int64 max_cas_entry_size_bytes = 8;
```

Comment:
s/entry/object/ maybe?

Looking at remote_execution.proto, it looks like 'entry' or 'cache entry' mainly refers to 'Action Cache entries'. For things in the CAS, we frequently seem to call them 'blobs' or 'objects'.

Reply:
I'm fine with blob. Changing it